### DISCUSSION TITLE
Add a `refresh` button to every page in mobile app

### DISCUSSION THREAD

**Discussion by vPumpking** [community/community#109164] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

When you want to refresh a page, for example an issue, you have to scroll all the way to the top of the page, which isn't convenient especially whe the amount of comments is huge.
I suggest adding a refresh button to any scrollable page, in the top right menu for example.
![Screenshot_20240219_131458_GitHub](https://github.com/community/community/assets/121762735/9e780657-71cf-40df-8d8f-cf8b5576dc27)

**Comment by iskae** [comment, marked as answer]
Hey @vPumpking 👋 

Thank you for reaching out and suggesting this ❤️ 

As part of our accessibility efforts, we have already started adding this 🎉 Please stay tuned for our future releases 😊 

Cheers!
